miso-web/src/spring-test/java/uk/ac/bbsrc/tgac/miso/webapp/springtest/AbstractST.java

```diff
@@ -109,7 +114,15 @@ private File getScript(String filename) {
     return script;
   }
 
-  public static String makeJson(Object obj) throws Exception {
+  public String makeJson(Object obj) throws Exception {
```

Comment:
Aside from the signature, this method is identical to makeJson(List<?> obj). Object is the parent class of all objects, including Lists, so this duplication should be unneeded, could you change the signature of the existing method instead?